packages/sheets-thread-comment/src/controllers/sheets-thread-comment.controller.ts

```diff
@@ -45,6 +45,8 @@ export const DefaultSheetsThreadCommentConfig: IUniverSheetsThreadCommentConfig
 
 @OnLifecycle(LifecycleStages.Starting, SheetsThreadCommentController)
 export class SheetsThreadCommentController extends Disposable {
+    private _setting = false;
```

Comment:
What is the flag used for? It deservers comments.

Reply:
Changed to `_isSwitchToCommenting` means is switching to an comment cell directly(maybe switch to a sub-sheet), avoid to be effect by selection init of sub-sheet if it was not initialized before